### ISSUE TITLE
Browser UIs fail when multiple tabs of UI in use

### ISSUE THREAD

**Issue by polarathene** [kaikreuzer/openhab-core#51]
When a browser has an openHAB UI open in more than one tab, errors are thrown and the UI fails to work/render properly. Issue was originally created at [`openhab/openhab`](https://github.com/openhab/openhab/issues/4488) but was redirected to this repo.

## Expected Behavior
UI should be possible to navigate with additional tabs.

## Current Behavior
I have heard this occurs in Paper UI and others but have not verified yet. HABmin(specifically v2) is confirmed by the developer to have this problem. The UI works fine in a single tab, or in multiple browsers with one tab each, but not when a browser opens an additional tab of that UI. 
- In HABmins case a modal appears stating the server is offline throwing many errors for file/state requests which breaks the rendering. 
- Refreshing the tab appears to make a connection successfully(refreshing again will put it back into server offline).
- More details covered with logs in [this HABmin issue](https://github.com/cdjackson/HABmin2/issues/183)

## Possible Solution
As it's isolated issue between browsers, it would seem browser related rather than anything going on on openHABs end? Possibly the browsers local storage if in use, or some sort of caching? I don't know much about Jetty(web server?) but I guess it could be caused there if this affects multiple/all UI.

## Steps to Reproduce (for bugs)
1. Open a browser and access a UI (I used HABmin 2)
2. Open a 2nd tab(I duplicated the tab/url on the configuration/bindings page)
3. Web console has now probably thrown a bunch of errors and UI is failing to load/work/unresponsive.

## Context
Not a major issue, I'm used to having multiple tabs, this actual issue appeared because I had over 100 tabs open and had an openHAB UI tab open that I was not aware of.

## Your Environment
Browsers tested with this issue: Firefox 47, Chrome 50, Chromium 50.
openHAB used: Latest OH2 Docker image(beta 3?) on an Arch Linux x86_64 host.
HABmin 2, 1.60-snapshot.
Add-ons: HUE, Sonos, KNX, Z-Wave. All installed via Paper UI.

**Comment by kaikreuzer** [comment]
Sorry having to redirect you again, but what you were told is not right.
Neither HABmin2 nor Paper UI are maintained here, the repos are either https://github.com/cdjackson/HABmin2/issues/ or https://github.com/eclipse/smarthome/issues.
I cannot clearly understand from your description whether it is about HABmin, Paper UI or both in conjunction. Your "steps to reproduce" point to HABmin2.

**Comment by polarathene** [comment]
It could potentially be Jetty if they're doing something the basic and classic are not? I'll forward the issue again, thanks :)

**Comment by polarathene** [comment]
@kaikreuzer I was trying to use the Classic UI to control an item in a sitemap for a KNX device. It was unresponsive, I had another UI open(HABmin) at the time, closing it then allowed all KNX events of me toggling the switch to fire, Classic UI became responsive to my toggles again. Opening the HABmin UI once more caused Classic UI to become unresponsive again and HABmin UI to fail working.

Further update, I had a Basic UI running as well. Further testing, HABmin2 seems to run fine with it's own instance(except for multiple tabs of HABmin2), but if both Basic and Classic UI are active it breaks, only one does not seem to affect it. When this happens both Basic and Classic UI are unresponsive as well, they seem to function if HABmin2 is not in the mix. I'm not sure if Paper UI provides access to a sitemap so I cannot confirm with that UI.

Update: They appear unresponsive, the switch command does go through and toggle the device but it takes between 15-20 seconds. Closing any of the UI tabs will make that instant again. Something is going on with the web server or the UIs that is blocking state changes issued by REST requests? I just confirmed this with a direct REST request via Postman, I imagine the same would happen via cURL.

I realize it's unlikely to run into this situation, although I've not confirmed if fully if it is isolated to browser tabs or something else. My concern is that multiple users with/without different clients(mobile apps ios/android, browser, etc) might trigger this. As the issue is not specific to a single browser or UI I think it is more likely something to do with the web server.
